docs/app/pages/components/components-sections/timeline/timeline/timeline.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="row">
+    <div class="col-md-10">
+        <button class="btn button-primary" (click)="addEvent()">Add New Event</button>
+        <ux-timeline>
+            <ux-timeline-event *ngFor="let ticket of tickets" [badgeColor]="ticket.color" [badgeTitle]="ticket.shortDate"
+                [title]="ticket.title" [timestamp]="ticket.date">
+        
+                <p>Ticket
+                    <a [href]="ticket.url">{{ticket.id}}</a>
+                    was {{ticket.status}} {{ticket.assignee}}.
+                </p>
+        
+            </ux-timeline-event>
+        </ux-timeline>
+    </div>
+</div>
+
+<hr>
+
+<p>The <code>ux-timeline</code> component is a responsive, data-driven vertical component to tell a story, show history or describe a sequence of events. Events are created with the <code>ux-timeline-event</code> component, which includes a customizable badge and informational panel.</p>
```

Comment:
Document that the content of the `ux-timeline-event` is rendered in the panel.

Reply:
Text added.